arangod/ClusterEngine/ClusterTransactionState.cpp

```diff
@@ -43,6 +43,7 @@
 #include "Transaction/ManagerFeature.h"
 #include "Transaction/Methods.h"
 #include "VocBase/LogicalCollection.h"
+#include "Utils/CollectionNameResolver.h"
```

Comment:
nit: convention is to sort includes alphabetically
```suggestion
#include "Utils/CollectionNameResolver.h"
#include "VocBase/LogicalCollection.h"
```